src/ServiceContainer/SymfonyExtension.php

```diff
@@ -99,18 +127,15 @@ public function initialize(ExtensionManager $extensionManager): void
     public function configure(ArrayNodeDefinition $builder): void
     {
         $builder
-            ->addDefaultsIfNotSet()
-                ->children()
-                    ->scalarNode('env_file')->defaultNull()->end()
-                    ->arrayNode('kernel')
-                        ->addDefaultsIfNotSet()
-                        ->children()
-                            ->scalarNode('bootstrap')->defaultValue('app/autoload.php')->end()
-                            ->scalarNode('path')->defaultValue('app/AppKernel.php')->end()
-                            ->scalarNode('class')->defaultValue('AppKernel')->end()
-                            ->scalarNode('env')->defaultValue('test')->end()
-                            ->booleanNode('debug')->defaultTrue()->end()
-                        ->end()
+            ->children()
+                ->scalarNode('env_file')->end()
+                ->arrayNode('kernel')
+                    ->children()
+                        ->scalarNode('bootstrap')->end()
+                        ->scalarNode('path')->end()
+                        ->scalarNode('class')->end()
+                        ->scalarNode('env')->end()
```

Comment:
default to `test` if omitted

Reply:
This default is defined in `SYMFONY_DEFAULTS` / `SYMFONY_4_DEFAULTS` -> `DEFAULT_ENV`. We can't set a default on input loading because we are losing information if given value has been entered by the user or is a default one (and we need that information in `autoconfigure` method).